<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pokemon Gen 3 Starter Manip</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_IV_Constrain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>IV Contstrain</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_Clear_Constrain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Neutral_Iv_Or_Not">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>         -                    Neutral                   +</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_HP_IV">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>80</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>HP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Attack_IV">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>117</y>
       <width>39</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Attack</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Defense_IV">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>150</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Defense</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Special_Attack_IV">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>190</y>
       <width>85</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Special Attack</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Special_Defense_IV">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>230</y>
       <width>97</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Special Defense</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Speed_IV">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>270</y>
       <width>38</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_HP_IV">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>71</y>
       <width>251</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Minu_Attack_IV">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>111</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Neutral_Attack_IV">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>111</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Plus_Attack_IV">
     <property name="geometry">
      <rect>
       <x>391</x>
       <y>111</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Minu_Defense_IV">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>151</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Neutral_Defense_IV">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>151</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Plus_Attack_IV_2">
     <property name="geometry">
      <rect>
       <x>391</x>
       <y>151</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Minus_Special_Attack_IV">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>191</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Neutral_Special_Attack_IV">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>191</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Plus_Special_Attack_IV">
     <property name="geometry">
      <rect>
       <x>391</x>
       <y>191</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Minus_Special_Defense_IV">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>231</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Neutral_Special_Defense_IV">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>231</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Plus_Special_Defense_IV">
     <property name="geometry">
      <rect>
       <x>391</x>
       <y>231</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Minus_Speed_IV">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>271</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Neutral_Speed_IV">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>271</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Plus_Speed_IV">
     <property name="geometry">
      <rect>
       <x>391</x>
       <y>271</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Nature">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Nature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="pushButton_Check_All_Nature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Check All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_UnCheck_All_Nature">
     <property name="geometry">
      <rect>
       <x>289</x>
       <y>310</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Uncheck All</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>551</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Hardy">
          <property name="text">
           <string>Hardy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Lonely">
          <property name="text">
           <string>Lonely</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Brave">
          <property name="text">
           <string>Brave</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Adamant">
          <property name="text">
           <string>Adamant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Naughty">
          <property name="text">
           <string>Naughty</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Bold">
          <property name="text">
           <string>Bold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Docile">
          <property name="text">
           <string>Docile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Relaxed">
          <property name="text">
           <string>Relaxed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Impish">
          <property name="text">
           <string>Impish</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Lax">
          <property name="text">
           <string>Lax</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Timid">
          <property name="text">
           <string>Timid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Hasty">
          <property name="text">
           <string>Hasty</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Serious">
          <property name="text">
           <string>Serious</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Jolly">
          <property name="text">
           <string>Jolly</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Naive">
          <property name="text">
           <string>Naive</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Modest">
          <property name="text">
           <string>Modest</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Mild">
          <property name="text">
           <string>Mild</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Quiet">
          <property name="text">
           <string>Quiet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Bashful">
          <property name="text">
           <string>Bashful</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Rash">
          <property name="text">
           <string>Rash</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Calm">
          <property name="text">
           <string>Calm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Gentle">
          <property name="text">
           <string>Gentle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Sassy">
          <property name="text">
           <string>Sassy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Careful">
          <property name="text">
           <string>Careful</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Nature_Quirky">
          <property name="text">
           <string>Quirky</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Starter_Pokemon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>481</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Starter Pokemon</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="pushButton_Search_Pokemon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Clear_TID">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>270</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear TID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Trainer_ID">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>238</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Max_Frame">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>206</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Minimum_Frame">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>174</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_Pokemon_Image">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_Pokemon_Image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Pokemon Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox_Pokemon_Names">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>142</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>140</y>
       <width>70</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_Pokemon_Name">
        <property name="text">
         <string>Pokemon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Min_Frame">
        <property name="text">
         <string>Min Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Max_Frame">
        <property name="text">
         <string>Max Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Trainer_ID">
        <property name="text">
         <string>Trainer ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Pokemon_Found_List">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>571</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Pokemon Found List</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QScrollBar" name="verticalScrollBar_Pokemon_Found_List">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>20</width>
       <height>231</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_Pokemon_Found">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>531</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Search_Around_Frame">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>680</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Around Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Change_Pokemon_Level">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>680</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 5</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
